src/NServiceBus.NHibernate/TimeoutPersisters/TimeoutEntityMap.cs

```diff
@@ -28,13 +28,13 @@ public TimeoutEntityMap()
                 pm.Length(1024);
             });
             Property(p => p.SagaId, pm => pm.Index("TimeoutEntity_SagaIdIdx"));
-            Property(p => p.Time, pm => pm.Index("TimeoutEntity_EndpointIdx"));
-            Property(p => p.Headers, pm => pm.Type(NHibernateUtil.StringClob));
             Property(p => p.Endpoint, pm =>
             {
                 pm.Index("TimeoutEntity_EndpointIdx");
                 pm.Length(440);
             });
+            Property(p => p.Time, pm => pm.Index("TimeoutEntity_EndpointIdx"));
```

Comment:
@MarcinHoppe Does this also affect index column order? If so, lets add a comment here too.